.azurepipelines/Platform-Build-VS.yml

```diff
@@ -28,8 +28,7 @@ jobs:
       QemuQ35_DEBUG:
         BuildPackage: QemuQ35Pkg
         BuildFile: "Platforms/QemuQ35Pkg/PlatformBuild.py"
-        BuildArch: IA32,X64
-        BuildFlags: ""
+        BuildFlags: "-a IA32,X64"

```

Comment:
I thought we were removing the build flags from platforms as it is not necessary ~ why add it back? I was under the impression we were going to remove the -a flag from QemuSbsa and QemuQ35.

Reply:
I don't mind the `-a` flag, but I'm fine with removing it. I'll update to do so.

Reply:
@Javagedes, can you please confirm this thread can be closed?